src/python/pants/core/goals/fmt_test.py

```diff
@@ -15,7 +15,7 @@
 from pants.engine.fs import EMPTY_DIGEST, CreateDigest, Digest, FileContent
```

Comment:
This file could use a good amount of TLC. It seems to under-test `fmt` pretty badly and has more than a fe quirks.

It's out of scope for this PR though. I cleaned it up enough to test the fix (fails before, passes after)